src/client/client/game.js

```diff
@@ -263,18 +289,30 @@ let game = function() {
 			_enteredMarbleList[marble.entryId] = marble;
 
 			// Add mesh
-			marbleManager.spawnMarble(marble);
+			let mesh = marbleManager.spawnMarble(marble);
```

Comment:
Maybe we should make the mesh a property of the marble object that is stored in `_enteredMarbleList`. It could be useful in the future (like accessing oob marbles directly?), and `_toggleMarbleTracking` will only need one argument which is a little cleaner.